spatial-lib/src/cmr/spatial/point.clj

```diff
@@ -138,6 +138,7 @@
   (equiv [this o]
          (and (isa? (class o) Point)
               (.equals this o)))
+  ;; The following functions need to be here because this type implements IPersistentMap, which inherits the functions.
```

Comment:
Remove this comment.

Reply:
I think the existing comment is saying we don't know why the error only shows up in CI, not in local testing.